homework02/sudoku.py

```diff
@@ -19,15 +22,11 @@ def create_grid(puzzle: str) -> tp.List[tp.List[str]]:
 
 
 def display(grid: tp.List[tp.List[str]]) -> None:
-    """Вывод Судоку """
+    """Вывод Судоку"""
     width = 2
     line = "+".join(["-" * (width * 3)] * 3)
     for row in range(9):
-        print(
-            "".join(
-                grid[row][col].center(width) + ("|" if str(col) in "25" else "") for col in range(9)
-            )
-        )
+        print("".join(grid[row][col].center(width) + ("|" if str(col) in "25" else "") for col in range(9)))
```

Comment:
По стандарту pep8 длина строчки кода должна быть не более 79 символов. Лучше будет перенести часть строки
```suggestion
        print("".join(grid[row][col].center(width) + 
                  ("|" if str(col) in "25" else "") for col in range(9)))
```